packages/angular/cli/models/schematic-command.ts

```diff
@@ -309,7 +308,8 @@ export abstract class SchematicCommand<
               question.type = 'confirm';
               break;
             case 'list':
-              question.type = 'list';
+            case 'checkbox':
+              question.type = definition.type;
```

Comment:
The initial plan for this concept was to model this as a `multiselect` question option on the list type itself. Could you make that change?